py/selenium/webdriver/remote/errorhandler.py

```diff
@@ -49,154 +51,136 @@
-    UNABLE_TO_SET_COOKIE = UnableToSetCookieException
-    UNEXPECTED_ALERT_OPEN = UnexpectedAlertPresentException
-    NO_ALERT_OPEN = NoAlertPresentException
-    SCRIPT_TIMEOUT = TimeoutException
-    IME_NOT_AVAILABLE = ImeNotAvailableException
-    IME_ENGINE_ACTIVATION_FAILED = ImeActivationFailedException
-    INVALID_SELECTOR = InvalidSelectorException
-    SESSION_NOT_CREATED = SessionNotCreatedException
-    MOVE_TARGET_OUT_OF_BOUNDS = MoveTargetOutOfBoundsException
-    INVALID_XPATH_SELECTOR = InvalidSelectorException
-    INVALID_XPATH_SELECTOR_RETURN_TYPER = InvalidSelectorException
-    ELEMENT_NOT_INTERACTABLE = ElementNotInteractableException
-    INSECURE_CERTIFICATE = InsecureCertificateException
-    INVALID_ARGUMENT = InvalidArgumentException
-    INVALID_COORDINATES = InvalidCoordinatesException
-    INVALID_SESSION_ID = InvalidSessionIdException
-    NO_SUCH_COOKIE = NoSuchCookieException
-    UNABLE_TO_CAPTURE_SCREEN = ScreenshotException
-    ELEMENT_CLICK_INTERCEPTED = ElementClickInterceptedException
-    UNKNOWN_METHOD = UnknownMethodException
+ERROR_TO_EXC_MAPPING: Dict[str, Type[WebDriverException]] = {
+    'element click intercepted': ElementClickInterceptedException,
+    'element not interactable': ElementNotInteractableException,
+    'insecure certificate': InsecureCertificateException,
+    'invalid argument': InvalidArgumentException,
+    'invalid cookie domain': InvalidCookieDomainException,
+    'invalid element state': InvalidElementStateException,
+    'invalid selector': InvalidSelectorException,
+    'invalid session id': InvalidSessionIdException,
+    'javascript error': JavascriptException,
+    'move target out of bounds': MoveTargetOutOfBoundsException,
+    'no such alert': NoAlertPresentException,
+    'no such cookie': NoSuchCookieException,
+    'no such element': NoSuchElementException,
+    'no such frame': NoSuchFrameException,
+    'no such window': NoSuchWindowException,
+    'no such shadow root': NoSuchShadowRootException,
+    'script timeout': TimeoutException,
+    'session not created': SessionNotCreatedException,
+    'stale element reference': StaleElementReferenceException,
+    'detached shadow root': NoSuchShadowRootException,
+    'timeout': TimeoutException,
+    'unable to set cookie': UnableToSetCookieException,
+    'unable to capture screen': ScreenshotException,
+    'unexpected alert open': UnexpectedAlertPresentException,
+    'unknown command': UnknownMethodException,
+    'unknown error': WebDriverException,
+    'unknown method': UnknownMethodException,
+    'unsupported operation': UnknownMethodException,
+    'element not visible': ElementNotVisibleException,
+    'element not selectable': ElementNotSelectableException,
+    'invalid coordinates': InvalidCoordinatesException,
+}
 
 
 class ErrorCode:
     """Error codes defined in the WebDriver wire protocol."""
 
     # Keep in sync with org.openqa.selenium.remote.ErrorCodes and errorcodes.h
     SUCCESS = 0
```

Comment:
We don't need to use the Error codes at all anymore

Reply:
Removed the ErrorCode class from errorhandler.py. in py/selenium/webdriver/remote/remote_connection.py, i have left 0 and 15 instead. The `status` probably safe to remove, but I'm not so confident to include the change in this PR since the `value` also has `value` (vanilla webdriver response) key. That may need to tweak properly at the same time, I think.